pages/dkp/kommander/2.2/DKP-insight/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Insight
+title: Insight
+excerpt: Use DKP insight to monitor and tune your enviornment
+menuWeight: 55
+beta: true
+enterprise: false
+---
+
+<p class="message--note"><strong>NOTE:</strong> Insights capability is offered with this release as a Technical preview only. Future enhancements include add-on licensing, a seamless user experience, and additional insight items.</p>
+
+DKP Insights is D2iQ’s latest analytics tool, designed to assist Kubernetes Administrators with daily tasks, such as making sure clusters are secure, debugging, monitoring, and anticipating future issues with your cluster. Insights provides you predictive analytics capability that provides greater support productivity, speed, and reduced cost. An Insights module, integrated into the DKP, collects configuration, logs, events, and metrics from DKP deployments and generates predictive insights on potential issues of varying critical levels so that you can quickly identify and resolve issues that would save a lot of time and money by not escalating to D2IQ's support.
```

Comment:
- 'Latest analytics tool' - do we have others?  
 - I don't understand what this sentence means:
 "Insights provides predictive analytics capability that provides greater support productivity, speed, and reduced cost."  
 Is any of that a direct customer benefit?   I suggest we remove this entire sentence.

```suggestion
DKP Insights is D2iQ’s latest analytics tool, designed to assist Kubernetes Administrators with daily tasks, such as ensuring your clusters are secure, debugging, monitoring, and anticipating issues. The Insights module, integrated into DKP, collects configuration, logs, events, and metrics from DKP deployments and generates predictive insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.
```

Reply:
1. Plz remove the latest from the "latest analytics tool" 
2. "designed to assist Kubernetes Administrators with daily tasks, such as making sure clusters are secure, debugging, monitoring, and anticipating future issues with your cluster. Insights provides you predictive analytics capability that provides greater support productivity, speed, and reduced cost" - This looks all convoluted and is not coming across well. Can we remove this?
3. "Insights module, integrated into the DKP," can be replaced with "Insights in DKP collects logs, events, and metrics from DKP management and managed k8s clusters and generates rules based insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.

Reply:
I think I added comments without opening the review link. Please find my comments